readme.md

```diff
@@ -1,7 +1,7 @@
 # yfshop
 this shop is intended for POWERUSERS.  
 do NOT except a simple to use interface.
-1. Run the installer script: `wget https://raw.githubusercontent.com/yourfriendoss/yfshop/main/installer.lua` then `installer.lua`
+1. Run the installer script: `wget run https://raw.githubusercontent.com/yourfriendoss/yfshop/main/installer.lua`
```

Comment:
This is incorrect, sorry for not mentioning you about it in Discord.
The installer is supposed to be ran on updates too, so I would not reccomend using `wget run`

Reply:
fixed in commit https://github.com/yourfriendoss/yfshop/pull/1/commits/d62f003c24c33140b1479ac15cf31d8a0475e214